rotten_tomatoes/correlation_analysis.py

```diff
@@ -0,0 +1,26 @@
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+class CorrelationAnalysis:
+    
+    def __init__(self, d):
+        self.data = d
+        
+    def corr_matrix(self, subset = None):
+        if subset == None:
```

Comment:
Nit: You should only have to use one "if" statement here. 
```
if subset: 
    return self.data[subset].corr()
   
return self.data.corr()
```

Reply:
Updated